kornia/geometry/homography.py

```diff
@@ -200,64 +245,62 @@ def sample_is_valid_for_homography(points1: Tensor, points2: Tensor) -> Tensor:
     return sample_is_valid
 
 
-def find_homography_lines_dlt(
-    lineseg_start1: Tensor,
-    lineseg_end1: Tensor,
-    lineseg_start2: Tensor,
-    lineseg_end2: Tensor,
-    weights: Optional[Tensor] = None,
-) -> Tensor:
+def find_homography_lines_dlt(ls1_: Tensor,
+                              ls2_: Tensor,
+                              weights: Optional[Tensor] = None) -> Tensor:
     r"""Compute the homography matrix using the DLT formulation for line correspondences.
-
     See :cite:`homolines2001` for details.
     The linear system is solved by using the Weighted Least Squares Solution for the 4 Line correspondences algorithm.
     Args:
-        lineseg_start1: A set of start points in the first image with a tensor shape :math:`(B, N, 2)`.
-        lineseg_end1: A set of end points in the first image with a tensor shape :math:`(B, N, 2)`.
-        lineseg_start2: A set of start points in the second image with a tensor shape :math:`(B, N, 2)`.
-        lineseg_end2: A set of end points in the second image with a tensor shape :math:`(B, N, 2)`.
+        ls1: A set of line segments in the first image with a tensor shape :math:`(B, N, 2, 2)`.
+        ls2: A set of line segments in the second image with a tensor shape :math:`(B, N, 2, 2)`.
         weights: Tensor containing the weights per point correspondence with a shape of :math:`(B, N)`.
     Returns:
         the computed homography matrix with shape :math:`(B, 3, 3)`.
     """
-    KORNIA_CHECK_SHAPE(lineseg_start1, ["B", "N", "2"])
-    KORNIA_CHECK_SHAPE(lineseg_end1, ["B", "N", "2"])
-    KORNIA_CHECK_SHAPE(lineseg_start2, ["B", "N", "2"])
-    KORNIA_CHECK_SHAPE(lineseg_end2, ["B", "N", "2"])
-
-    device, dtype = _extract_device_dtype([lineseg_start1, lineseg_start2])
+    if len(ls1_.shape) == 3:
+        ls1: Tensor = ls1_[None]
```

Comment:
```suggestion
        ls1 = ls1[None]
```
why not this ? to avoid underscore in the signature

Reply:
Wouldn't it modify the argument shape as well?